### PR TITLE
Adding Contributing guidelines 

### PR DESCRIPTION
**Pull request by priyansh19**
It contains more diversed contributing guidelines

### CODE REVIEW

File: CONTRIBUTION.md [hunk 1]
```diff
@@ -0,0 +1,16 @@
+# :memo: Contributing
+
+To start contributing to [Whatsapp-Play](https://github.com/rpotter12/whatsapp-play), please first discuss the change you wish to make via creating a issue 
+in the issue section or any other method with the owners of this repository before making a change. 
+
+We have a code of conduct, please follow it in all your interactions with the project.
+
+# :rocket: Steps for creating your first PR
+
+1. Ensure any install or build dependencies are checked and verified before the end when releasing a build.<br />
```
Comment (rpotter12) [inline CONTRIBUTION.md]: Codacy Issue found: [[list-item-indent] Incorrect list-item indent: add 1 space](https://app.codacy.com/manual/rpotter12/whatsapp-play/pullRequest?prid=5115816)

**Comment by rpotter12** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
- Added 6
           
```


See the complete overview on [Codacy](https://app.codacy.com/manual/rpotter12/whatsapp-play/pullRequest?prid=5115816&bid=16818452)

File: CONTRIBUTION.md [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+
+2. Update the README.md with details of changes including environment variables, various file parameters and container details **if needed**.<br />
```
Comment (rpotter12) [inline CONTRIBUTION.md]: Codacy Issue found: [[list-item-indent] Incorrect list-item indent: add 1 space](https://app.codacy.com/manual/rpotter12/whatsapp-play/pullRequest?prid=5115816)

File: CONTRIBUTION.md [hunk 3]
```diff
@@ -0,0 +1,16 @@
...
+
+3. Try adding screenshots in your pull request of the changes you made.
+
+4. You may be able to merge the pull request in once you have the sign-off of two other developers, or if you do not have permission to do that, you may request the maintainer or reviewer to merge it for you.
```
Comment (rpotter12) [inline CONTRIBUTION.md]: Codacy Issue found: [[list-item-indent] Incorrect list-item indent: add 1 space](https://app.codacy.com/manual/rpotter12/whatsapp-play/pullRequest?prid=5115816)

File: CONTRIBUTION.md [hunk 4]
```diff
@@ -0,0 +1,16 @@
+# :memo: Contributing
+
+To start contributing to [Whatsapp-Play](https://github.com/rpotter12/whatsapp-play), please first discuss the change you wish to make via creating a issue 
+in the issue section or any other method with the owners of this repository before making a change. 
+
+We have a code of conduct, please follow it in all your interactions with the project.
+
+# :rocket: Steps for creating your first PR
```
Comment (rpotter12) [inline CONTRIBUTION.md]: Codacy Issue found: [[no-multiple-toplevel-headings] Don’t use multiple top level headings (1:1)](https://app.codacy.com/manual/rpotter12/whatsapp-play/pullRequest?prid=5115816)

File: CONTRIBUTION.md [hunk 5]
```diff
@@ -0,0 +1,16 @@
...
+
+1. Ensure any install or build dependencies are checked and verified before the end when releasing a build.<br />
+
+2. Update the README.md with details of changes including environment variables, various file parameters and container details **if needed**.<br />
+
+3. Try adding screenshots in your pull request of the changes you made.
+
```
Comment (rpotter12) [inline CONTRIBUTION.md]: Codacy Issue found: [[list-item-spacing] Extraneous new line after list item](https://app.codacy.com/manual/rpotter12/whatsapp-play/pullRequest?prid=5115816)

File: CONTRIBUTION.md [hunk 6]
```diff
@@ -0,0 +1,16 @@
...
+We have a code of conduct, please follow it in all your interactions with the project.
+
+# :rocket: Steps for creating your first PR
+
+1. Ensure any install or build dependencies are checked and verified before the end when releasing a build.<br />
+
+2. Update the README.md with details of changes including environment variables, various file parameters and container details **if needed**.<br />
+
+3. Try adding screenshots in your pull request of the changes you made.
```
Comment (rpotter12) [inline CONTRIBUTION.md]: Codacy Issue found: [[list-item-indent] Incorrect list-item indent: add 1 space](https://app.codacy.com/manual/rpotter12/whatsapp-play/pullRequest?prid=5115816)

**Comment by rpotter12** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
- Added 6
           
```


See the complete overview on [Codacy](https://app.codacy.com/manual/rpotter12/whatsapp-play/pullRequest?prid=5115816&bid=16818452)

File: CONTRIBUTION.md (hunk 1 above)
Comment (rpotter12) [inline CONTRIBUTION.md]: Codacy Issue found: [[list-item-indent] Incorrect list-item indent: add 1 space](https://app.codacy.com/manual/rpotter12/whatsapp-play/pullRequest?prid=5115816)

File: CONTRIBUTION.md (hunk 6 above)
Comment (rpotter12) [inline CONTRIBUTION.md]: Codacy Issue found: [[list-item-indent] Incorrect list-item indent: add 1 space](https://app.codacy.com/manual/rpotter12/whatsapp-play/pullRequest?prid=5115816)

File: CONTRIBUTION.md (hunk 4 above)
Comment (rpotter12) [inline CONTRIBUTION.md]: Codacy Issue found: [[no-multiple-toplevel-headings] Don’t use multiple top level headings (1:1)](https://app.codacy.com/manual/rpotter12/whatsapp-play/pullRequest?prid=5115816)

File: CONTRIBUTION.md (hunk 2 above)
Comment (rpotter12) [inline CONTRIBUTION.md]: Codacy Issue found: [[list-item-indent] Incorrect list-item indent: add 1 space](https://app.codacy.com/manual/rpotter12/whatsapp-play/pullRequest?prid=5115816)

File: CONTRIBUTION.md (hunk 5 above)
Comment (rpotter12) [inline CONTRIBUTION.md]: Codacy Issue found: [[list-item-spacing] Extraneous new line after list item](https://app.codacy.com/manual/rpotter12/whatsapp-play/pullRequest?prid=5115816)

File: CONTRIBUTION.md (hunk 3 above)
Comment (rpotter12) [inline CONTRIBUTION.md]: Codacy Issue found: [[list-item-indent] Incorrect list-item indent: add 1 space](https://app.codacy.com/manual/rpotter12/whatsapp-play/pullRequest?prid=5115816)

**Comment by rpotter12** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
- Added 9
           
```


See the complete overview on [Codacy](https://app.codacy.com/manual/rpotter12/whatsapp-play/pullRequest?prid=5115816&bid=16818452)

File: CONTRIBUTION.md [hunk 7]
```diff
@@ -0,0 +1,16 @@
...
+To start contributing to [Whatsapp-Play](https://github.com/rpotter12/whatsapp-play), please first discuss the change you wish to make via creating a issue 
+in the issue section or any other method with the owners of this repository before making a change. 
+
+We have a code of conduct, please follow it in all your interactions with the project.
+
+# :rocket: Steps for creating your first PR
+
+1. Ensure any install or build dependencies are checked and verified before the end when releasing a build.<br />
+
+1. Update the README.md with details of changes including environment variables, various file parameters and container details **if needed**.<br />
```
Comment (rpotter12) [inline CONTRIBUTION.md]: Codacy Issue found: [[list-item-indent] Incorrect list-item indent: add 1 space](https://app.codacy.com/manual/rpotter12/whatsapp-play/pullRequest?prid=5115816)

File: CONTRIBUTION.md [hunk 8]
```diff
@@ -0,0 +1,16 @@
...
+
+1. Try adding screenshots in your pull request of the changes you made.
```
Comment (rpotter12) [inline CONTRIBUTION.md]: Codacy Issue found: [[list-item-indent] Incorrect list-item indent: add 1 space](https://app.codacy.com/manual/rpotter12/whatsapp-play/pullRequest?prid=5115816)

File: CONTRIBUTION.md (hunk 4 above)
Comment (rpotter12) [inline CONTRIBUTION.md]: Codacy Issue found: [[no-multiple-toplevel-headings] Don’t use multiple top level headings (1:1)](https://app.codacy.com/manual/rpotter12/whatsapp-play/pullRequest?prid=5115816)

File: CONTRIBUTION.md [hunk 9]
```diff
@@ -0,0 +1,16 @@
...
+
+1. You may be able to merge the pull request in once you have the sign-off of two other developers, or if you do not have permission to do that, you may request the maintainer or reviewer to merge it for you.
```
Comment (rpotter12) [inline CONTRIBUTION.md]: Codacy Issue found: [[ordered-list-marker-value] Marker should be `4`, was `1`](https://app.codacy.com/manual/rpotter12/whatsapp-play/pullRequest?prid=5115816)

File: CONTRIBUTION.md [hunk 10]
```diff
@@ -0,0 +1,16 @@
...
+We have a code of conduct, please follow it in all your interactions with the project.
+
+# :rocket: Steps for creating your first PR
+
+1. Ensure any install or build dependencies are checked and verified before the end when releasing a build.<br />
+
+1. Update the README.md with details of changes including environment variables, various file parameters and container details **if needed**.<br />
+
+1. Try adding screenshots in your pull request of the changes you made.
+
```
Comment (rpotter12) [inline CONTRIBUTION.md]: Codacy Issue found: [[list-item-spacing] Extraneous new line after list item](https://app.codacy.com/manual/rpotter12/whatsapp-play/pullRequest?prid=5115816)

File: CONTRIBUTION.md (hunk 1 above)
Comment (rpotter12) [inline CONTRIBUTION.md]: Codacy Issue found: [[list-item-indent] Incorrect list-item indent: add 1 space](https://app.codacy.com/manual/rpotter12/whatsapp-play/pullRequest?prid=5115816)

File: CONTRIBUTION.md (hunk 8 above)
Comment (rpotter12) [inline CONTRIBUTION.md]: Codacy Issue found: [[ordered-list-marker-value] Marker should be `3`, was `1`](https://app.codacy.com/manual/rpotter12/whatsapp-play/pullRequest?prid=5115816)

File: CONTRIBUTION.md (hunk 9 above)
Comment (rpotter12) [inline CONTRIBUTION.md]: Codacy Issue found: [[list-item-indent] Incorrect list-item indent: add 1 space](https://app.codacy.com/manual/rpotter12/whatsapp-play/pullRequest?prid=5115816)

File: CONTRIBUTION.md (hunk 7 above)
Comment (rpotter12) [inline CONTRIBUTION.md]: Codacy Issue found: [[ordered-list-marker-value] Marker should be `2`, was `1`](https://app.codacy.com/manual/rpotter12/whatsapp-play/pullRequest?prid=5115816)

**Comment by xandao6** [comment]
https://github.com/rpotter12/whatsapp-play/issues/142 , this issue is assigned

**Pull request closed without merging**